openmetadata-ui/src/main/resources/ui/src/pages/ServiceDetailsPage/ServiceMainTabContent.tsx

```diff
@@ -69,14 +83,18 @@ function ServiceMainTabContent({
   serviceDetails,
   saveUpdatedServiceData,
   pagingInfo,
+  isVersionPage = false,
 }: Readonly<ServiceMainTabContentProps>) {
   const { t } = useTranslation();
   const { serviceCategory } = useParams<{
     serviceCategory: ServiceTypes;
   }>();
 
   const { fqn: serviceFQN } = useFqn();
+  const { permissions } = usePermissionProvider();
+
   const [isEdit, setIsEdit] = useState(false);
+  const [pageData, setPageData] = useState<ServicePageData[]>([]);
```

Comment:
Same here, please avoid using a different local state to store the same data in the child.

Reply:
Updated the code.